frontend/public/components/chargeback.tsx

```diff
@@ -146,13 +126,14 @@ const DataCell = ({name, value, maxValue, total}) => {
   return value;
 };
 
-const DataTable = ({rows, orderBy, sortBy, applySort, keys, maxValues, totals}:DataTableProps) => {
+const DataTable = ({rows, orderBy, sortBy, applySort, keys}:DataTableProps) => {
   const size = _.clamp(Math.floor(12 / _.size(rows[0])), 1, 4);
   const className = `col-md-${size}`;
   return <div className="co-m-table-grid co-m-table-grid--bordered" style={{marginTop: 20, marginLeft: -15, marginRight: -15}}>
     <ListHeader>
       {_.map(keys, k => <ColHead
-        className={classNames(className, {'text-right': REDUCER_COLS.indexOf(k) < 0})}
+        className={classNames(className, {'text-right': _.isFinite(_.get(rows, [0, k]))})}
+        wrap={true} // chargeback report header titles can be long
```

Comment:
Let's always wrap headers instead of adding a new prop. (Although again this will conflict with @priley86's PR.)

Reply:
`<ColHead ...` has/had a hardcoded `text-nowrap`.  I added a flag to not use it for Chargeback reports.  Are you saying remove `text-nowrap` from `<ColHead...>`  -this will affect many views which also use `ColHead`.

Reply:
Right, I'm saying remove `text-nowrap` from `ColHead`. It will affect all uses.

Reply:
Regardless, this will all change with #1465. I'd avoid a special prop for now since we'd have to change it for PF4 tables anyway. I assume they handle headers wrapping OK as-is, @priley86 ?

Reply:
Fixed, removed `text-nowrap`